src/plugins/fs/ext.c

```diff
@@ -237,6 +237,26 @@ void bd_fs_ext4_info_free (BDFSExt4Info *data) {
     bd_fs_ext2_info_free ((BDFSExt2Info*) data);
 }
 
+BDExtraArg __attribute__ ((visibility ("hidden")))
+**bd_fs_ext2_mkfs_options (BDFsMkfsOptions *options) {
```

Comment:
I made this function "private" but it can also be public. I'm not sure if it would be useful or not.